sdk/storage/azure-storage-common/inc/azure/storage/common/version.hpp

```diff
@@ -14,7 +14,7 @@
 #define AZURE_STORAGE_COMMON_VERSION_MAJOR 12
 #define AZURE_STORAGE_COMMON_VERSION_MINOR 0
 #define AZURE_STORAGE_COMMON_VERSION_PATCH 0
-#define AZURE_STORAGE_COMMON_VERSION_PRERELEASE "beta.4"
```

Comment:
Wasn't the last release of storage-common beta.3?
https://github.com/Azure/azure-sdk-for-cpp/releases/tag/azure-storage-common_1.0.0-beta.3
https://azure.github.io/azure-sdk-for-cpp/storage.html

Are we skipping 4 to synchronize all the libraries to a single version?

Reply:
Yes, if this is after GA release we won't do this. But since it is a private preview, I think it would be nice for us to tell the customer to fetch beta.5 version, intead of 'fetch blob with 12.0.0-beta.5 and common with 12.0.0-beta.4'.

Reply:
And because we already have a leap from `1.0.0` to `12.0.0`, I don't find this approach concerning.

Reply:
Agreed.